src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+```bash
+mkdir nonprod && cd nonprod
+touch terragrunt.hcl
+```
+
+In the  nonprod terragrunt.hcl file copy over the configuration from the environment __terragrunt.hcl__:
+```hcl
+include {
+	path= find_in_parent_folders()
+}
+
+terraform {
+	source = "../../src"
+}
+
+inputs = {
+	newrelic_personal_apikey = "NRAK-***" # Your New Relic account ID
+	newrelic_account_id = "12345" # Your New Relic account ID
+	newrelic_region = "US" # US or EU (defaults to US)
+}
+```
+
+<Callout variant="tip">
+If you are using a different account for your nonprod environment, you can update the inputs with a new New Relic account ID and API key.
+</Callout>
+
+Inside of your __nonprod__ directory, run `terragrunt init` then run `terragrunt apply`.
+
+In the output, you will see Terraform has created a new set of resources. The new resources should have the nonprod name prefixed.
+
+</Step>
+
+<Step>
+
+</Step>
+Terragrunt has created two environments, but there is no difference between them other than the name.
+
+Adding variables in the configurations allows your configurations to be more dynamic. In the next steps, you will add variables to your configurations.
+
+In the __main.tf__ add new variables for the Host Conditions modules cpu_critical, cpu_warning, and diskPercentage values:
+```hcl
+variable "cpu_critical" {default = 89}
+variable "cpu_warningl" {default = 79}
+variable "diskPercentage" {default = 69}
+```
+
+In the __main.tf__ update the HostConditions module values to use the cpu_critical, cpu_warning, and diskPercentage variables:
+```hcl
+module "HostConditions" {
+	 source = "git::https://github.com/jsbnr/demo-terraform.git"
+	 policyId = newrelic_alert_policy.DemoPolicy.id
+	 cpu_critical = var.cpu_critical
+	 cpu_warning = var.cpu_warninig
+	 diskPercent = var.dskPercentage
+	}
+```
+Run `terragrunt plan`. In the output, the HostConditions values have changed to use the variable defaults.
+
+Next, you want to pass the values into your environment configurations by adding the inputs block variables. In the __nonprod terrgrunt.hcl__ fiile, add the cpu_critical, cpu_warning, and diskPercentage variables:
+```hcl
+inputs = {
+	newrelic_personal_apikey = "NRAK-***" # Your New Relic account ID
+	newrelic_account_id = "12345" # Your New Relic account ID
+	newrelic_region = "US" # US or EU (defaults to US)
+
+	cpu_critical = 50
+	cpu_warninig = 40
+	diskPercentage = 98
+}
+```
+
+Run `terragrunt apply`. In your New Relic account, you will see a new policy with configurations changed based on the environment.
+</Steps>
+
+## Conclusion
+
+Congratulations! You've used Terragrunt to generate your New Relic configurations and managed multiple environments.  Review the [New Relic Terraform provider documentation](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs) and [Terragrunt quick start](https://terragrunt.gruntwork.io/docs/getting-started/quick-start/) to learn how you can take your configuration to the next level.
```

Comment:
Can we add a link to my example project too perhaps? https://github.com/jsbnr/nr-terragrunt-intro-example